File: dataset/ZeroShot/CLAUDE_claude-sonnet-4-5-20250929/Task10.py

def get_sum(a, b):
    if a == b:
        return a
    
    min_val = min(a, b)
    max_val = max(a, b)
    
    # Using arithmetic progression formula: sum = n * (first + last) / 2
    n = max_val - min_val + 1
    return n * (min_val + max_val) // 2


if __name__ == "__main__":
    # Test case 1
    print(f"Test 1 (1, 0): {get_sum(1, 0)} (Expected: 1)")
    
    # Test case 2
    print(f"Test 2 (1, 2): {get_sum(1, 2)} (Expected: 3)")
    
    # Test case 3
    print(f"Test 3 (0, 1): {get_sum(0, 1)} (Expected: 1)")
    
    # Test case 4
    print(f"Test 4 (1, 1): {get_sum(1, 1)} (Expected: 1)")
    
    # Test case 5
    print(f"Test 5 (-1, 2): {get_sum(-1, 2)} (Expected: 2)")

File: dataset/ZeroShot/CLAUDE_claude-sonnet-4-5-20250929/Task10.c

#include <stdio.h>

int getSum(int a, int b) {
    if (a == b) {
        return a;
    }
    
    int minVal = (a < b) ? a : b;
    int maxVal = (a > b) ? a : b;
    
    // Using arithmetic progression formula: sum = n * (first + last) / 2
    int n = maxVal - minVal + 1;
    return n * (minVal + maxVal) / 2;
}

int main() {
    // Test case 1
    printf("Test 1 (1, 0): %d (Expected: 1)\\n", getSum(1, 0));
    
    // Test case 2
    printf("Test 2 (1, 2): %d (Expected: 3)\\n", getSum(1, 2));
    
    // Test case 3
    printf("Test 3 (0, 1): %d (Expected: 1)\\n", getSum(0, 1));
    
    // Test case 4
    printf("Test 4 (1, 1): %d (Expected: 1)\\n", getSum(1, 1));
    
    // Test case 5
    printf("Test 5 (-1, 2): %d (Expected: 2)\\n", getSum(-1, 2));
    
    return 0;
}
